README.md

```diff
@@ -13,24 +13,24 @@
 ├── .gitignore
 |
 ├── data
-│   ├── demofile.json
-│   └── testfile.json
+│   ├── demofile.json
+│   └── testfile.json
 |
 ├── scripts
-│   ├── cogitate.sh
-│   └── test.sh
+│   ├── cogitate.sh
+│   └── test.sh
 ├── src
-│   ├── cogitate.py
-│   ├── data_collection.py
-│   ├── data_processor.py
-│   ├── driller.py
+│   ├── cogitate.py
+│   ├── data_collection.py
+│   ├── data_processor.py
+│   ├── driller.py
 │   ├── json_handler.py
```

Comment:
The current file structure no longer has json_handler either, this readme should probably be updated to reflect any changes that were made to the overall file structure.

Reply:
I agree that the README should be updated, other teams are currently working on that and I don't want our changes to cause conflicts. As for the json_handler, it does, in fact, still exist in the master branch. I use it in the data_collection module and removing it will cause many issues.

Reply:
Ah, my bad. I was under the impression that #69 was merged already, as it aims to merge data_collection and json_handler

Reply:
No worries, I personally prefer keeping the two modules because `data_collection` is becoming very large and harder to manage and read.